web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/DatastreamRestControllerIT.java

```diff
@@ -183,53 +204,57 @@ public void testGetMultipleDatastreams() throws Exception {
     @Test
     public void testGetThumbnail() throws Exception {
         var corpus = populateCorpus();
-
         PID filePid = corpus.pid6File;
         String id = filePid.getId();
-        createDerivative(id, THUMBNAIL_SMALL, BINARY_CONTENT.getBytes());
 
-        MvcResult result = mvc.perform(get("/thumb/" + filePid.getId()))
+        var filename = "bunny.jpg";
+        var formattedBasePath = "/iiif/v3/" + ImageServerUtil.getImageServerEncodedId(id);
+        stubFor(WireMock.get(urlMatching(formattedBasePath + "/full/!64,64/0/default.jpg"))
+                .willReturn(aResponse()
+                        .withStatus(HttpStatus.OK.value())
+                        .withBody(filename)
+                        .withHeader("Content-Type", "image/jpeg")));
+
+        MvcResult result = mvc.perform(get("/thumb/" + id))
                 .andExpect(status().is2xxSuccessful())
                 .andReturn();
 
         // Verify content was retrieved
         MockHttpServletResponse response = result.getResponse();
-        assertEquals(BINARY_CONTENT, response.getContentAsString());
-        assertEquals(BINARY_CONTENT.length(), response.getContentLength());
-        assertEquals("image/png", response.getContentType());
-        assertEquals("inline; filename=\"" + id + "." + THUMBNAIL_SMALL.getExtension() + "\"",
-                response.getHeader(CONTENT_DISPOSITION));
+        // TO DO assert correct image returned
+        assertEquals("image/jpeg", response.getContentType());
+        assertEquals("attachment; filename=file_64px.jpg", response.getHeader(CONTENT_DISPOSITION));
```

Comment:
Previously the thumbnail wasn't being returned as an attachment (which triggers a "download" prompt in browsers), and I think we want to keep the old behavior. So we might have to update downloadImageService.streamImage to take a boolean to tell it to do attach mode or not